### PR TITLE
update Client Authentication Methods readability

### PR DESCRIPTION
**Pull request by Dzuming**
- update Client Authentication Methods readability

### CODE REVIEW

File: kibana.md [hunk 1]
```diff
@@ -1125,7 +1125,13 @@ readonlyrest_kbn.auth:
 
 ### Client Authentication Methods
 
-You can configure how the ReadonlyREST Kibana plugin sends `client_id` and `client_secret` to the identity provider using the option `readonlyrest_kbn.auth.<YOUR_OIDC_CONFIG>.tokenEndpointAuthMethod`. There are two available methods for authentication:
+You can configure how the ReadonlyREST Kibana plugin sends `client_id` and `client_secret` to the identity provider using the option 
+
+```
+readonlyrest_kbn.auth.oidc_kc.tokenEndpointAuthMethod: 'client_secret_post' // Available options: client_secret_basic (default) or client_secret_post
```
Comment (sscarduzio) [inline kibana.md:1131]: yaml has # as a comment

File: kibana.md [hunk 2]
```diff
@@ -1125,7 +1125,13 @@ readonlyrest_kbn.auth:
 
 ### Client Authentication Methods
 
-You can configure how the ReadonlyREST Kibana plugin sends `client_id` and `client_secret` to the identity provider using the option `readonlyrest_kbn.auth.<YOUR_OIDC_CONFIG>.tokenEndpointAuthMethod`. There are two available methods for authentication:
+You can configure how the ReadonlyREST Kibana plugin sends `client_id` and `client_secret` to the identity provider using the option 
+
+```
```
Comment (coutoPL) [inline kibana.md:1130]: let's add "```yaml"

File: kibana.md (hunk 1 above)
Comment (coutoPL) [inline kibana.md:1131]: maybe it'd be better to use the "dot-less" syntax?
```yaml
readonlyrest_kbn:
  auth:
    oidc_kc:
      tokenEndpointAuthMethod: 'client_secret_post' // Available options: client_secret_basic (default) or client_secret_post
      [...]
```

to show that this is not a separate setting, but belong to some group of settings

Comment (Dzuming) [reply kibana.md]: Done ✔️

Comment (Dzuming) [reply kibana.md]: Sounds good, changed ✔️

File: kibana.md (hunk 2 above)
Comment (Dzuming) [reply kibana.md]: Done ✔️

**Review by coutoPL — APPROVED** [review]
LGTM

**Pull request merged**